apps/next-evans/components/core/BookTeaserCard.tsx

```diff
@@ -11,8 +11,12 @@ import { LANG } from '@/lib/env';
 import { isCompilations } from '@/lib/friend';
 import Button from '@/components/core/Button';
 import { mostModernEdition } from '@/lib/editions';
+import { editionTypes } from '@/lib/document';
 
-export type Props = DocumentWithMeta & {
+export type Props = Omit<
```

Comment:
hmm... i know in the past, when picking between `Pick<T>` and `Omit<T>`, i've used the rule of thumb of just preferring whichever ended up being shorter.  

but i'm wondering if for these prop types, if maybe we should take a hard stance that we always use `Pick`. it's more explicit, doesn't accidentally randomly grow when we query new things from the db, and would reduce some diff churn, at the expense of in some places being a bit more verbose.  what do you think?

Reply:
but.. i wonder if we'd end up with suuuuper long types.  waffling.